components/xsd-fu/xsd-fu

```diff
@@ -190,26 +192,6 @@ def main(model, opts):
             content = content.render(encoding='utf-8')
             customAsXMLElementPropertyContent[propertyName] = content
         # END Custom templates for asXMLElment()
-        parents = model.resolve_parents(obj.name)
```

Comment:
Ah, OK.  In that case this was removed for a few reasons.  The cmake build captures stdout to introspect what's being generated, and enabling debugging breaks things horribly by spewing rubbish.  Debugging is now restricted to additional information expanded in the templates.  Additionally, the debug output is mostly just a copy of what's in the actual generated code, so writing it to a file and also to stdout isn't giving us anything we don't already have.  For the structured annoations, this is for debugging a single feature; this is included as a more general property list in the C++ templates, and the same can also be added to the Java templates if desired.